cmd/launcher/interactive.go

```diff
@@ -80,7 +80,7 @@ func runInteractive(args []string) error {
 		flOsqueryFlags = append(flOsqueryFlags, fmt.Sprintf("tls_server_certs=%s", certs))
 	}
 
-	osqueryProc, extensionsServer, err := interactive.StartProcess(logger, rootDir, osquerydPath, flOsqueryFlags)
+	osqueryProc, extensionsServer, err := interactive.StartProcess(rootDir, osquerydPath, flOsqueryFlags)
```

Comment:
Should we be passing a slogger here? It feels weird to not have logs?

Reply:
Yeah I think I need to adjust this, just haven't been sure how to configure the slogger for command-line yet -- I'll get it sorted!